.github/workflows/build.yaml

```diff
@@ -40,34 +41,33 @@ jobs:
 
       # Generate .vsix package for the extension 
       - name: Generate .vsix package
-        if: runner.os == 'Linux'  
```

Comment:
no there's no need to generate the vsix 3 times. So only linux generates and uploads the archive